apps/docs/src/app/core/component-docs/action-sheet/examples/action-sheet-mobile/action-sheet-mobile-example.component.html

```diff
@@ -9,8 +9,8 @@
             <li fd-action-sheet-item label="Download" glyph="download" (click)="actionPicked('download')"></li>
             <li fd-action-sheet-item label="Accept" glyph="accept" (click)="actionPicked('accept')"></li>
             <li fd-action-sheet-item label="Reject" glyph="cancel" (click)="actionPicked('cancel')"></li>
-            <li fd-action-sheet-item label="Decide later" [isCloseButton]="true"></li>
-            <li fd-action-sheet-item label="Cancel" [isCloseButton]="true"></li>
+            <li fd-action-sheet-item label="Decide Later" [isCloseButton]="true"></li>
+            <li fd-action-sheet-item label="Cancel" negative="true" [isCloseButton]="true"></li>
```

Comment:
Just for clarification: for providing the boolean value we need to use square brackets `[negative]="true"`, without the brackets value will be parsed as string